lint-workflow/lint.py

```diff
@@ -190,10 +344,24 @@ def main():
                     input_files.append(filepath)
     else:
         print("File/Directory does not exist, exiting.")
+        return -1
```

Comment:
I believe return codes in POSIX are force to be unsigned. This might be the same as a return code of `1` on a UNIX machine.

Reply:
TBH I used exit codes a lot during my uni projects, and I was taught that `-1` is an exit code when the user did sth wrong, but maybe it's only a Linux thing ;)

Reply:
I vaguely remember `-1` exit codes from `main()` in C++ from uni. POSIX is the the standard portable operating system interfaces for *NIX systems, so it could very well be just a *NIX thing. Windows might do the `-1`. I know that I'm biased towards POSIX since I'm a Linux user. I'm good with leaving the `-1` as the general "user did something wrong" exit code". I think POSIX will handle it just by forcing it to be unsigned on a Linux runner.